webapp/channels/src/packages/mattermost-redux/src/utils/remote_cluster_utils.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {DateTime, Interval} from 'luxon';
+
+import type {RemoteCluster} from '@mattermost/types/remote_clusters';
+
+const SiteURLPendingPrefix = 'pending_';
+export const isConfirmed = (rc: RemoteCluster) => Boolean(rc.site_url && !rc.site_url.startsWith(SiteURLPendingPrefix));
+export const isConnected = (rc: RemoteCluster) => {
```

Comment:
I'd suggest changing this to something like `isOnline` to keep consistency with the server

Reply:
isConnected is the previous naming on the client, so I'll use it unless we do a bigger refactoring.